app/src/main/java/de/danoeh/antennapod/activity/AudioplayerActivity.java

```diff
@@ -569,6 +569,8 @@ protected void onPlaybackSpeedChange() {
 
     private void updateButPlaybackSpeed() {
         if (controller != null && controller.canSetPlaybackSpeed()) {
+            //Feed feed = <must somehow get feed here!!>
+            //butPlaybackSpeed.setText(feed.getPreferences().getCurrentPlaybackSpeed());
```

Comment:
This does not help me - the ```controller.getCurrentPlaybackSpeedMultiplier()``` method returns the speed multiplier of what's *currenty* being played - I want the speed multiplier of the current feed, as defined by the new ```FeedPreferences.getPlaybackSpeed()``` method.
In order to get it, I need the Feed class instance of the currently playing podcast.